spec/features/publish/accepting_rollover_recruitment_spec.rb

```diff
@@ -16,7 +16,7 @@
   scenario "i can accept the rollover interruption" do
     then_i_am_taken_to_the_rollover_recruitment_page
     when_i_accept_rollover_recruitment
-    then_i_should_be_returned_to_the_publish_service_page
+    then_i_should_be_redirected_to_the_providers_show_page
```

Comment:
So i fixed these specs in an earlier merge to make them more predictable. Are you able to rebase to those changes? I'm not sure why this particular change seems to fix it. Since it should be returning them to the publish root path eg: `/publish/organisations` as the spec starts with that visit before they're interrupted?

Reply:
I think it is working as intended. The `public/organisations` index redirects to the show action for a single provider, which I think you have set up the test with?
```
def index
      authorize :provider, :index?

      page = (params[:page] || 1).to_i
      per_page = 10
      @pagy, @providers = pagy(providers.order(:provider_name), page: page, items: per_page)

      render "publish/providers/no_providers", status: :forbidden if @providers.blank?
      redirect_to publish_provider_path(@providers.first.provider_code) if @providers.count == 1
    end
```

Reply:
Ah! Ok sounds good